src/include/kdb.h.in

```diff
@@ -45,6 +45,23 @@ extern "C" {
 }
 #endif
 
+#define KDB_GET_PHASE_RESOLVER "resolver"
```

Comment:
Please move to plugin interface.

Prefix with ELEKTRA_ (but can be done in later PRs).